src/app/router/routes.ts

```diff
@@ -0,0 +1,9 @@
+// PATHs names react-router is matched to in path={}
+export const PATHS = {
+  lnurlPay: "lnurlPay",
+} as const;
+
+// URLs which should match the router PATHs to call `navigate(URL)`
+export const URLS = {
+  lnurlPay: `/${PATHS.lnurlPay}`,
```

Comment:
why do we have these URLS a and PATHS objects?

with that we write `path={PATHS.lnurlPay}` instead of `path="lnurlPay"` in the popup and prompt?

Reply:
Correct, the string "lnurlpay" is always the same.
This is only the beginning of a CONST collection to avoid repeating the same strings everywhere when creating `<Routes>` or when navigating with react-router 
and to import all PATHS later to the TS types to let TS know which routes name exist.

Reply:
but what's the difference to writing `PATHS.lnurlPay` instead of `lnurlPay` ? either I repeat a getter or a string. 
I don't care URLS + PATHS sounds complex to me (at least this right now puts `??` in my head)